integration-tests/verification-flow/range-proofs.test.ts

```diff
@@ -49,94 +51,45 @@ describe('Range proofs verification', () => {
-  //       id: 'b8acdd33-e8e0-491f-8674-6ba521d9ea84',
-  //       name: 'test data',
-  //       nonce: '798b999c18bb83f0f9669aac91c68910',
-  //       created: '2025-01-15T16:34:06.077Z',
-  //       response_url:
-  //         'https://api-testnet.truvera.io/proof-requests/b8acdd33-e8e0-491f-8674-6ba521d9ea84/send-presentation',
-  //       request: {
-  //         id: 'b8acdd33-e8e0-491f-8674-6ba521d9ea84',
-  //         input_descriptors: [
-  //           {
-  //             id: 'Credential 1',
-  //             name: 'test data',
-  //             purpose: 'test',
-  //             constraints: {
-  //               fields: [
-  //                 {
-  //                   path: ['$.credentialSubject.id'],
-  //                   optional: false,
-  //                 },
-  //                 {
-  //                   path: ['$.credentialSubject.number'],
-  //                   filter: {
-  //                     type: 'number',
-  //                     exclusiveMinimum: 0,
-  //                   },
-  //                   optional: false,
-  //                   predicate: 'required',
-  //                 },
-  //                 {
-  //                   path: ['$.credentialSchema.id'],
-  //                   filter: {
-  //                     const:
-  //                       'https://schema.truvera.io/TestingCheqdSchema-V1-1736958567072.json',
-  //                   },
-  //                 },
-  //                 {
-  //                   path: ['$.credentialSubject.name'],
-  //                   optional: false,
-  //                 },
-  //               ],
-  //             },
-  //           },
-  //         ],
-  //       },
-  //       type: 'proof-request',
-  //     },
-  //   });
-
-  //   const attributesToReveal = [
-  //     'credentialSubject.id',
-  //     'credentialSubject.name',
-  //     'credentialSubject.date',
-  //     'credentialSubject.number',
-  //   ];
-
-  //   controller.selectedCredentials.set(credential.id, {
-  //     credential: credential,
-  //     attributesToReveal,
-  //   });
-
-  //   const presentation = await controller.createPresentation();
-
-  //   expect(
-  //     presentation.verifiableCredential[0].credentialSubject,
-  //   ).toBeDefined();
-  // });
+  it('should not reveal range proof attributes for KVAC credentials', async () => {
+    const wallet: IWallet = await getWallet();
+    const controller = await createVerificationController({
+      wallet,
+    });
+
+    const credentialUrl =
+      'https://creds-testnet.truvera.io/16c431993e6678af4fc74a9b995250bfe3a2ecf215baa31f0945164ac89fd798';
+    const password = 'test';
+    const {data: credential} = await axios.get(
+      `${credentialUrl}?p=${btoa(password)}`,
+    );
+
+    getCredentialProvider().addCredential(credential);
 
+    await controller.start({
+      template: proofRequest.qr,
+    });
+
+    // Reveal only dateOfBirth attribute
+    const attributesToReveal = [
```

Comment:
is having the word, "Reveal" here confusing if we're trying to prove that the attribute isn't actually revealed?

Reply:
Good point, as range proof attribute will not be revealed in the presentation we could skip this part. pexToBounds will define which range proof attributes should be included, and any range proof attributes added to this attributesToReveal array will be removed automatically

Reply:
I've updated the tests and also added better assertions for range proof bounds